src/share/streams_nuopc/dshr_strdata_mod.F90

```diff
@@ -71,13 +70,14 @@ module dshr_strdata_mod
      type(ESMF_RouteHandle)              :: routehandle                     ! stream n -> model mesh mapping
      character(CS), allocatable          :: fldlist_stream(:)               ! names of stream file fields
      character(CS), allocatable          :: fldlist_model(:)                ! names of stream model fields
-     type(ESMF_FieldBundle)              :: fldbun_stream_lb                ! stream n field bundle for lb of time period (stream grid)
-     type(ESMF_FieldBundle)              :: fldbun_stream_ub                ! stream n field bundle for ub of time period (stream grid)
-     type(ESMF_FieldBundle)              :: fldbun_model_lb                 ! stream n field bundle for lb of time period (model grid)
-     type(ESMF_FieldBundle)              :: fldbun_model_ub                 ! stream n field bundle for ub of time period (model grid)
-     type(ESMF_FieldBundle)              :: fldbun_model                    ! stream n field bundle for model time (model grid)
-     integer                             :: ustrm                           ! index of vector u in stream
-     integer                             :: vstrm                           ! index of vector v in stream
+     integer                             :: stream_lb
```

Comment:
Please add a comment to what this is. It would be good to have every attribute in these derived types commented.